airflow/operators/dagrun_operator.py

```diff
@@ -38,7 +38,8 @@ class TriggerDagRunLink(BaseOperatorLink):
     name = 'Triggered DAG'
 
     def get_link(self, operator, dttm):
-        return f"/graph?dag_id={operator.trigger_dag_id}&root=&execution_date={quote(dttm.isoformat())}"
+        query = {"dag_id": operator.trigger_dag_id, "root": "", "execution_date": dttm.isoformat()}
+        return build_airflow_url_with_query(query)
```

Comment:
Can you use url_for here? I guess it should work.

Reply:
This will work only if inside of application context and I'm not sure if this is necessary
```
RuntimeError: Attempted to generate a URL without the application context being pushed. This has to be executed when application context is available.
```

Reply:
It seems to me that this link will be generated by the webserver, so this is expected behavior.

Reply:
Please see https://github.com/apache/airflow/pull/11778#discussion_r515906811